qiskit/circuit/quantumcircuit.py

```diff
@@ -560,8 +560,8 @@ def qasm(self):
 
     def draw(self, scale=0.7, filename=None, style=None, output=None,
              interactive=False, line_length=None, plot_barriers=True,
-             reverse_bits=False, justify=None, idle_wires=True, vertical_compression='medium',
-             with_layout=True, fold=None):
+             reverse_bits=False, justify=None, vertical_compression='medium', idle_wires=True,
```

Comment:
I just checked the source of this diff and it looks #3108 accidentally changed this (my rebase reverted it back to the previous order). See the last release: https://github.com/Qiskit/qiskit-terra/blob/0.9.0/qiskit/circuit/quantumcircuit.py#L534 

This actually fixes a potentially breaking change because the order of kwargs is part of the api. (unless we specified a `**` to indicate keyword only, which we can't introduce for the same backwards compat concerns)